src/sys/mman.rs

```diff
@@ -440,6 +439,34 @@ pub unsafe fn mmap<F: AsFd>(
     }
 }
 
+/// Create an anonymous memory mapping.
+///
+/// This function is a wrapper around [`mmap`]:
+/// `mmap(ptr, len, prot, MAP_ANONYMOUS | flags, -1, 0)`.
+///
+/// # Safety
+///
+/// See the [`mmap(2)`] man page for detailed requirements.
+///
+/// [`mmap(2)`]: https://man7.org/linux/man-pages/man2/mmap.2.html
+pub unsafe fn mmap_anonymous(
```

Comment:
Test code that uses `mmap()` for anonymous maping should be replaced with this new function:

```shell
$ rg "mmap::<"
src/sys/mman.rs
528:///     let mem = mmap::<BorrowedFd>(None, one_k_non_zero, ProtFlags::PROT_NONE,

test/sys/test_mman.rs
7:        let ptr = mmap::<BorrowedFd>(
32:        let mem = mmap::<BorrowedFd>(
90:        let mem = mmap::<BorrowedFd>(
```